### DISCUSSION TITLE
max dev 如何移除控制台info,以及加载动画

### DISCUSSION THREAD

**Discussion by SEELELancelot** [umijs/umi#12388] [Q&A]
max dev 如何移除控制台info,以及加载动画

info  - [你知道吗？] 默认使用 esbuild 作为 JavaScript 压缩工具，也可通过 jsMinifier 配置项切换到 terser 或 uglifyJs 等，详见 https://umijs.org/docs/api/config#jsminifier-webpack       
info  - Umi v4.2.1
info  - Preparing...

![1](https://github.com/umijs/umi/assets/36724474/c2366348-919f-4cb0-aad9-5af665841a18)

**Comment by fz6m** [comment]
关闭你知道了吗？

```bash
  DID_YOU_KNOW=none max dev
```

你说的加载动画是哪个，终端没有加载动画，在编译时打开网页的 umi 加载动画不可关闭。

**Comment by SEELELancelot** [comment]
谢谢
